database-seeds/collections/songs-iidx.json

```diff
@@ -17682,7 +17682,9 @@
 			"genre": "UPLIFTING ETHNIC"
 		},
 		"id": 1607,
-		"searchTerms": [],
+		"searchTerms": [
+			"Coma"
```

Comment:
Nice, were the search terms for older songs based on their ticker text or manually entered?

Reply:
Pulled from ticker text whenever it didn't match the full title, with a couple of manual fixups also (like Coma and Kaijin).